SimKev2/DrawParty
# Added Favicon

###### Feature:
A rudimentary favicon.ico 

###### Acceptance Criteria:
A favicon exists that is loaded with the page when it is served. 

###### Design Notes:
It's really bad if you look at it in full resolution but as a favicon it looks fine. 

###### How To Test:
1. Serve the page locally with `make serve`
2. navigate to localhost:8000 and verify that the favicon appears on the page
3. Verify that the server logs do not contain a "Missing file: <xxx>/favicon.ico" error

###### My Test Results:
There is no missing favicon error in the serve logs.
![image](https://user-images.githubusercontent.com/13294560/31855025-f2e37b9c-b667-11e7-8e3b-77b48d0aa6ca.png)


###### Reviewers: @JessicaLi1396 @NAWilson9
